Dockerfile

```diff
@@ -7,7 +7,7 @@
 ### ### ### ### ### ### ### ### ###
 # Builder layer
 
-FROM node:8.9.4-alpine as ghost-builder
+FROM alpine:latest as ghost-builder
```

Comment:
But with this one you don't have node, right?

Reply:
Yes, so I have to install it via `apk add`, which is 8.9.3,

Reply:
I think we have just try to fix the base problem without changing the base image :)
It is a user or access security problem... I'm checking it.

Note: If you want I add you as contributor or the master repository... you can create the branch directly here to propose the pull requests ;)

Reply:
FYI I'm testing this:
```
RUN mkdir /home/node/.npm-global
ENV PATH=/home/node/.npm-global/bin:$PATH
ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
```
It seems that just changing the npm base folder is fixing the problem

Reply:
Here you are the Docker file which is working locally on my Mac:
```
RG GHOST_VER="1.21.1"
FROM node:8.9.4-alpine as ghost-builder

ENV GHOST_VERSION=${GHOST_VERSION}                              \
    GHOST_INSTALL="/var/lib/ghost"                              \
    GHOST_CONTENT="/var/lib/ghost/content"                      \
    GHOST_USER="node"

# Set default directory
WORKDIR $GHOST_INSTALL

#Changing NPM/NODE Base folder
RUN mkdir /home/node/.npm-global && chown -R node:node /home/node/.npm-global
ENV PATH=/home/node/.npm-global/bin:$PATH
ENV NPM_CONFIG_PREFIX=/home/node/.npm-global

# We use SQLite as our DB. Force install "sqlite3" manually since it's an optional dependency of "ghost"
RUN set -eux                                                                            && \
    apk update && apk add su-exec make python                                           && \
    echo "---             S P A C E R             ---"                                  && \
    su-exec node npm install --loglevel=info -g ghost-cli                               && \
    echo "---             S P A C E R             ---"                                  && \
    chown node:node "$GHOST_INSTALL"                                                    && \
    su-exec node ghost install "$GHOST_VERSION" \
        --db sqlite3 --no-prompt                \
        --no-stack --no-setup                   \
        --dir "$GHOST_INSTALL"                                  && \
    echo "---             S P A C E R             ---"          && \
    su-exec node ghost config --ip 0.0.0.0      \
        --port 2368 --no-prompt --db sqlite3    \
        --url http://localhost:2368             \
        --dbpath "$GHOST_CONTENT/data/ghost.db"                 && \
    echo "---             S P A C E R             ---"          && \
    su-exec node ghost config paths.contentPath "$GHOST_CONTENT";
```
I let you test and deliver in your PullRequest, so we can accept your changes :)

Reply:
I think we can improve it with a split between operation we have to execute as root user (apk commands) and all others we can now execute as `node`.
In this way we can even prevent the usage of su-exec and start the node section with a:
```
USER node
```

Reply:
@mmornati We should do that in another PR since this would involve a refactor of the Dockerfile.